extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -61,4 +61,21 @@ public static function onArticleSaveComplete(
 
 		return true;
 	}
+
+	/**
+	 * Add community page entry point to article page right rail module
+	 *
+	 * @param array $railModuleList
+	 * @return bool
+	 */
+	public static function onGetRailModuleList( Array &$railModuleList ) {
+		$context = RequestContext::getMain();
+
+		if ( ( $context->getUser()->isLoggedIn() && $context->getTitle()->inNamespace( NS_MAIN ) ) ||
+			   $context->getTitle()->isSpecial( 'WikiActivity' ) ) {
```

Comment:
Strange, might need to double check that. I guess we might be considering anons who visit WikiActivity to be interested in contributing?